config/manager/network_policy.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-egress-to-api-server-dns-and-kubelet
+  namespace: system
+spec:
+  podSelector:
+    matchLabels:
+      control-plane: controller-manager
+  policyTypes:
+    - Egress
+  egress:
+    - to:
+      - namespaceSelector:
+          matchLabels:
+            name: kube-system
+        podSelector:
+          matchLabels:
+            k8s-app: kube-apiserver-proxy
+      ports:
```

Comment:
Are the ports needed if you already use the pod/namespace selectors?

Reply:
Also here I agree with David's  comment 
If using namespaces selector ports are required ?

Reply:
I wanted to be precise, but maybe its not needed, I shall remove the ports